packages/get-platform/src/getPlatform.ts

```diff
@@ -359,38 +371,29 @@ export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetO
   ])
 
   if (libsslFilename) {
-    debug(`Found libssl.so file using "ldconfig" or other generic paths: ${libsslFilenameFromSpecificPath}`)
+    debug(`Found libssl.so file using "ldconfig" or other generic paths: ${libsslFilename}`)
     const libsslVersion = parseLibSSLVersion(libsslFilename)
     if (libsslVersion) {
-      return libsslVersion
+      return { libssl: libsslVersion, strategy: 'ldconfig' }
     }
   }
 
+  /* Reading the libssl.so version didn't work, fall back to openssl */
+
   const openSSLVersionLine: string | undefined = await getFirstSuccessfulExec(['openssl version -v'])
 
   if (openSSLVersionLine) {
     debug(`Found openssl binary with version: ${openSSLVersionLine}`)
     const openSSLVersion = parseOpenSSLVersion(openSSLVersionLine)
     debug(`The parsed openssl version is: ${openSSLVersion}`)
     if (openSSLVersion) {
-      return openSSLVersion
-    }
-  }
-
-  /* Reading the libssl.so version didn't work, fall back to openssl */
-
-  const openSSLVersion: string | undefined = await getFirstSuccessfulExec(['openssl version -v'])
-
-  if (openSSLVersion) {
-    const matchedVersion = parseOpenSSLVersion(openSSLVersion)
-    if (matchedVersion) {
-      return matchedVersion
+      return { libssl: openSSLVersion, strategy: 'openssl-binary' }
     }
   }
```

Comment:
This block here was duplicated. Perhaps something that escaped our attention during a merge conflict resolution.